config/sitemap.rb

```diff
@@ -0,0 +1,14 @@
+Sitemap::Generator.instance.load(host: 'era.library.ualberta.ca') do
+  path :root, priority: 1, change_frequency: 'weekly'
+  path :catalog_index, priority: 1, change_frequency: 'weekly'
+  User.all.each do |user|
```

Comment:
Since we're getting out of the user portfolio business, do we want to feature user home pages in the Google sitemap? @sfbetz